src/Jackett.Common/Indexers/HDTorrents.cs

```diff
@@ -144,21 +142,29 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                 };
                 var hasFreeleech = freeleechRanks.Contains(rank);
 
-                var rows = dom.QuerySelectorAll(".mainblockcontenttt > tbody > tr:has(a[href^=\"details.php?id=\"])");
+                var rows = dom.QuerySelectorAll("table.mainblockcontenttt tr:has(td.mainblockcontent)").Skip(1);
                 foreach (var row in rows)
                 {
                     var release = new ReleaseInfo();
 
-                    release.Title = row.QuerySelector("td.mainblockcontent b a").TextContent;
-                    release.Description = row.QuerySelector("td:nth-child(3) > span").TextContent;
+                    var mainLink = row.Children[2].QuerySelector("a");
+
+                    release.Title = row.Children[2].QuerySelector("a").TextContent;
+                    release.Description = row.Children[2].QuerySelector("span").TextContent;
 
                     release.MinimumRatio = 1;
                     release.MinimumSeedTime = 172800; // 48 hours
 
                     var tdIndex = 0;
+
+                    //Maybe replace with something like this in the future? Can't test this
+                    //var editRow = row.Children.FirstOrDefault(node => string.Equals(
+                    //                                              node.TextContent, "Edit",
+                    //                                              StringComparison.OrdinalIgnoreCase));
+                    //var index = row.Children.Index(editRow);
                     if (row.QuerySelector("td:nth-last-child(1)").TextContent == "Edit")
                         tdIndex = 1;
-                    // moderators get additional delete, recomend and like links
+                    // moderators get additional delete, recommend and like links
```

Comment:
I think you can use `row.Children[X]` like in other cases. If the number of columns is fixed, just put the number, if not you can do something like `row.Children[row.Children.Length - X]`

Reply:
In the end you have made many changes to the indexes of the columns. I hope it is tested. :)

Reply:
This is tested and working